### PR TITLE
fix: remove spurious second argument from BatchNorm2d in GoogleNet

### PR DESCRIPTION
**Pull request by Mukller**
## Bug Fix: Spurious second argument in BatchNorm2d (GoogleNet b3 branch)

In the GoogleNet inception module (branch b3), `BatchNorm2d` is called with two
positional arguments: `nn.BatchNorm2d(n5x5, n5x5)`.

`nn.BatchNorm2d(num_features, eps=1e-05, ...)` — the second positional argument
is `eps` (epsilon for numerical stability). Passing `n5x5` (a channel count integer
like 32 or 64) as `eps` sets an incorrect epsilon value.

**Before (wrong):**
```python
nn.BatchNorm2d(n5x5, n5x5),   # ❌ n5x5 used as eps
```

**After (fixed):**
```python
nn.BatchNorm2d(n5x5),          # ✓ default eps=1e-05
```

All other `BatchNorm2d` calls in this file use a single argument (correct).
Reported in issue #96.

### CODE REVIEW

**Review by Mukller — COMMENTED** [review]
## Code Review: fix spurious second argument in BatchNorm2d

### Summary
Removes an erroneous second positional argument from a `BatchNorm2d` call in the GoogLeNet Inception module. `nn.BatchNorm2d(num_features, eps, ...)` interprets the second positional argument as `eps` (numerical stability epsilon), not as a second `num_features`. Passing `n5x5` (a channel count, likely 96 or similar) as `eps` would result in a drastically oversized epsilon, breaking batch normalization.

### Critical Issues
| # | File | Line | Issue | Severity |
|---|------|------|-------|----------|
| 1 | `models/googlenet.py` | ~45 | `nn.BatchNorm2d(n5x5, n5x5)` passes `n5x5` as both `num_features` and `eps` | 🔴 Critical |

### What the Fix Does
```python
# Before (bug)
nn.BatchNorm2d(n5x5, n5x5)  # n5x5 interpreted as eps — e.g. eps=96

# After (correct)
nn.BatchNorm2d(n5x5)  # uses default eps=1e-05
```

An `eps` value of 96 (or any typical channel count) would dominate the variance normalization, effectively making the layer output near-constant — silently degrading training.

### What Looks Good
- One-line fix with clear correctness rationale
- All other `BatchNorm2d` calls in the file use the single-argument form

### Verdict
**Request Changes → now fixed.** A silent training bug that would prevent the affected Inception branch from learning effectively.
